packages/module/src/ChatbotFooter/ChatbotFootnote.tsx

```diff
@@ -32,7 +32,7 @@ export interface ChatbotFootnotePopover {
   bannerImage?: ChatbotFootnotePopoverBannerImage;
   /** Optional CTA button that can be used to trigger an action and close the popover */
   cta?: ChatbotFootnotePopoverCTA;
-  /** Optional link that can be used to show and external link like **Learn More** */
+  /** Optional link that can be used to show and external link like **View AI policy** */
```

Comment:
```suggestion
  /** Optional link that can be used to show an external link like **View AI policy** */
```

This is not you - but I am noticing a typo in here. Would it be ok to fix it here?